### DISCUSSION TITLE
CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#74] [Announcements]
# CS自学指南

CS自学指南

https://csdiy.wiki/

**Comment by luoxiang1** [comment]
资源都非常不错，谢谢你的辛苦付出！

**Reply by zixiu214** [reply]
好人一生平安

**Comment by Windy147** [comment]
so good,谢谢！

**Comment by cxw620** [comment]
很棒, 难得有这么好的自学指南, 感谢你的付出~

**Comment by CNS1mple** [comment]
如果大一看到就好了...谢谢您的付出

**Comment by beyondlesliezl** [comment]
谢谢你，等我大学毕业了，如果您同意，我想和您面基，请您吃饭。

**Comment by alexzzkk** [comment]
感谢～

**Comment by MongoRolls** [comment]
很感谢你的分享,我收获到了一些东西.
对我来说,全英学习是最大的障碍.
来自于一名普通学校的大学生

**Comment by hakic** [comment]
非常感谢分享。有些课程我上个好几个，这上面提到的也是我觉得最好的选择！

请问博弈论是哪本书/哪个课程？我没找到。有拓扑的推荐资源吗？

**Reply by PKUFlyingPig** [reply]
耶鲁大学的，B站有搬运视频。拓扑我没有系统性地学过，但3Blue1Brown的视频里有不少拓扑的科普。

**Reply by hakic** [reply]
谢谢。我找找看

**Comment by venslymoker** [comment]
非常感谢您的付出，我已经将这个发给了我的许多朋友，一起学习的感觉真的太爽了！！！

**Comment by hekang-hk** [comment]
您的工作值得赞赏，为后来的人们照亮了道路！

**Comment by TURING132** [comment]
想请问一下61A的视频怎么观看呢？在youtube里面都显示private，无法观看

**Reply by PKUFlyingPig** [reply]
本书CS61A页面下方的评论有相关讨论可以参考

**Comment by Aizenz** [comment]
牛逼啊大哥，我大一的时候被数算和数分劝退了（PKU基础课质量真的堪忧），大二转系搞金融去了，兜兜转转发现自己还是有一个成为工程师的梦想！！现在业余学一学快乐自己

**Comment by huangtangsan** [comment]
好人一生平安

**Comment by Mendnet** [comment]
没有密码学吗 ？Dan Boneh的课B站上有，他自己也有本书；另外一本Introduction to Modern Crytography第一版有中译。

**Reply by PKUFlyingPig** [reply]
Dan Boneh的课本书有收录，见“一个仅供参考的CS学习规划”。

**Reply by PKUFlyingPig** [reply]
Introduction to Modern Crytography 在“好书推荐”有收录。

**Reply by Mendnet** [reply]
猛的

**Comment by momoknight** [comment]
这几年一大感触就是感谢开源

**Comment by codefun1024** [comment]
感谢作者！
